UselessMine.cpp

```diff
@@ -227,16 +221,28 @@ void UselessMine::Init (const char* commandLine)
 
     // Save the location of the file so we can reload after
     deathMessagesFile = commandLine;
-
+    defusalMessagesFile = commandLine;
```

Comment:
I think I fixed it. Lines 225 - 240 should probably be reviewed. Let me know if I need to fix it.